OnboardingMarsV4/Hooks/Hooks1.cs

```diff
@@ -0,0 +1,39 @@
+﻿using MarsOnboardV2.Drivers;
+using OpenQA.Selenium.Chrome;
+using MarsOnboardV2.Pages;
+using NUnit.Framework;
+using System.Xml.Linq;
+
+
+namespace MarsOnboardV2.Hooks
+{
+
+    [Binding]
+    public sealed class Hooks1 : CommonDriver
```

Comment:
Curious to know do you get multiple browsers opening?

Reply:
Every time at the beginning of the test browser opens and at the end of test it closes.